### DISCUSSION TITLE
Windows Installer

### DISCUSSION THREAD

**Discussion by joewilly11** [zwave-js/zwave-js-ui#3978] [General]
Is there any chance there could be a version of the zwave-js-ui that would auto-install on Windows without someone knowing how to code?  It's probably contradictory to the concept of open source but it's a utility I'd like to have.  Would be willing to pay $$.

**Comment by kpine** [comment]
You can just download the "win.zip" file from the releases, extract the  `.exe` file and run it. No installer is needed nor is any coding experience required.  https://github.com/zwave-js/zwave-js-ui/releases

You're on your own setting it up as a Windows Service though. You can find an example solution here: https://github.com/zwave-js/zwave-js-ui/discussions/3017#discussioncomment-5407562
